src/SettingsForm.tsx

```diff
@@ -49,7 +51,18 @@ export const SettingsForm = () => {
   return (
     <form>
       <label>
-        Page Width (in)
+        Unit
+        <select
+          disabled={isRendering}
+          value={settings.unit}
+          onChange={handleChange("unit", "value")}
```

Comment:
'value' is the default, so it's not required to pass here.
```suggestion
          onChange={handleChange("unit")}
```